mne/beamformer/_dics.py

```diff
@@ -446,35 +889,65 @@ def tf_dics(epochs, forward, noise_csds, tmin, tmax, tstep, win_lengths,
-    subtract_evoked : bool
-        If True, subtract the averaged evoked response prior to computing the
-        tf source grid.
-    mode : str
-        Spectrum estimation mode can be either: 'multitaper' or 'fourier'.
+        Only used in 'multitaper' or 'fourier' mode. For 'cwt_morlet' mode, use
+        the ``frequencies`` parameter instead.
+    frequencies : list of float | list of lists of float
+        The frequencies to compute the source power for. If you want to compute
+        the average power for multiple frequency bins, specify a list of
+        lists: each list containing the frequencies for the corresponding bin.
+        Only used in 'cwt_morlet' mode. In other modes, use the ``freq_bins``
+        parameter instead.
     n_ffts : list | None
-        FFT lengths to use for each frequency bin.
+        Length of the FFT for each frequency bin. If ``None`` (the default),
+        the exact number of samples between ``tmin`` and ``tmax`` will be used.
+        Only used in 'multitaper' or 'fourier' mode.
     mt_bandwidths : list of float
         The bandwidths of the multitaper windowing function in Hz. Only used in
         'multitaper' mode. One value should be provided for each frequency bin.
+        Defaults to None.
     mt_adaptive : bool
         Use adaptive weights to combine the tapered spectra into CSD. Only used
-        in 'multitaper' mode.
+        in 'multitaper' mode. Defaults to False.
     mt_low_bias : bool
         Only use tapers with more than 90% spectral concentration within
-        bandwidth. Only used in 'multitaper' mode.
+        bandwidth. Only used in 'multitaper' mode. Defaults to True.
+    cwt_n_cycles: float | list of float | None
+        Number of cycles to use when constructing Morlet wavelets. Fixed number
+        or one per frequency. Defaults to 7.
+        Only used in 'cwt_morlet' mode.
+    decim : int | slice
+        To reduce memory usage, decimation factor during time-frequency
+        decomposition. Defaults to 1 (no decimation).
+        Only used in 'cwt_morlet' mode.
+
+        If `int`, uses tfr[..., ::decim].
+        If `slice`, uses tfr[..., decim].
+    subtract_evoked : bool
+        If True, subtract the averaged evoked response prior to computing the
+        tf source grid. Defaults to False.
     reg : float
-        The regularization for the cross-spectral density.
+        Regularization to use for the DICS beamformer computation.
+        Defaults to 0.05.
     label : Label | None
-        Restricts the solution to a given label.
-    pick_ori : None | 'normal'
-        If 'normal', rather than pooling the orientations by taking the norm,
-        only the radial component is kept.
+        Restricts the solution to a given label. Defaults to None.
+    pick_ori : None | 'normal' | 'max-power'
+        The source orientation to estimate source power for:
+
+            ``None`` : orientations are pooled. (Default)
```

Comment:
These do not look right

https://6675-1301584-gh.circle-artifacts.com/0/tmp/circle-artifacts.U1ejhTC/html/generated/mne.beamformer.tf_dics.html#mne.beamformer.tf_dics